src/lib/include/libsed.h

```diff
@@ -13,11 +13,16 @@
 
 #define SED_MAX_KEY_LEN (256)
 
+enum SED_LOCK_SHIFT {
+	SED_RO_SHIFT = 0,
+	SED_RW_SHIFT = 1,
+	SED_LK_SHIFT = 2,
+};
+
 enum SED_LOCK_TYPE {
-	SED_NO_LOCK = 0x00,
-	SED_READ_LOCK = 0x01,
-	SED_WRITE_LOCK = 0x02,
-	SED_READ_WRITE_LOCK = (SED_READ_LOCK | SED_WRITE_LOCK),
+	SED_READ_LOCK = 1 << SED_RO_SHIFT,
```

Comment:
This is confusing to me. I read SED_RO_SHIFT as SED Read-only shift and then it is used to define value for SED READ_LOCK. Read-Only meand that read_lock is set to 0 and write lock is set to 1.

Reply:
Here I'm trying to define by how many bits does 1 need to be shifted in order to define the lock
eg; SED_RO_SHIIFT = 0;
SED_READ_LOCK = 1 << 0; -> SED_READ_LOCK = 1 and so on

Reply:
I understand bit shift operations. Chosen naming is confusing. Read-Only basically set Write-lock not Read-Lock

Reply:
Can you please suggest a good naming convention for this...

Reply:
Something like SED_RO_ACCESS, SED_RW_ACCESS, SED_NO_ACCESS